### ISSUE TITLE
Letsencrypt-Wildcards are not requested when used in main-domain (kv-store)

### ISSUE THREAD

**Issue by eBeyond** [containous/traefik#3092]
### Do you want to request a *feature* or report a *bug*?
bug

### What did you do?

I defined my domains in a key value store:

```
traefik/acme/domains/0/main: *.my-domain.de
traefik/acme/domains/0/sans/0: my-domain.de
```

### What did you expect to see?
I would expect that a certificate for that will be requested. The wildcard-domain in main would be the main the one in sans would be the alternative name for the first one. (Letsencrypt doesn't support wildcards in SANS)

### What did you see instead?
```
** time="2018-03-28T05:48:27Z" level=error msg="Error validating ACME certificate for domain []: unable to generate a wildcard certificate for domain \"*.my-domain.de,my-domain.de\" : SANs are not allowed"**
```

I would think that the main domain is also prepended to the sans.

### Output of `traefik version`: (_What version of Traefik are you using?_)
1.6.0-rc2-alpine

**Comment by nmengin** [comment]
Hello @eBeyond.

Many thanks for your interest in the project.

Today, Træfik does not allow generating wildcard certificates with SANs as it's described in the [documentation](https://docs.traefik.io/v1.6/configuration/acme/#wildcard-domain) : 
> Wildcard domains can currently be provided only by to the acme.domains option. Theses domains can not have SANs.

Even if we are currenly working to implement a solution which allows this feature, it's not as easy as it appears and it can take few times.

For the moment, as a workaround, WDYT to declare 2 domains with no SANs in your configuration?

**Comment by traefiker** [comment]
Closed by #3167.
